### PR TITLE
Add missing variable declarations

### PR DESCRIPTION
**Pull request by stweil**
This fixes several error messages from LGTM.

Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by stweil** [comment]
This pull request **fixes 8 alerts** when merging dc26190d039061fa45c6c21b2e7a4895f676a48f into 1f7191d752654ac79bdba9e5f98dca813ffd169e - [view on LGTM.com](https://lgtm.com/projects/g/UB-Mannheim/ocr-gt-tools/rev/pr-392d245cd2f46fb8789fed38042c709aaa011c66)

**fixed alerts:**

* 8 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: js/model/line.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function Line(opts) {
-    for (key in opts) { this[key] = opts[key]; }
+    for (var key in opts) { this[key] = opts[key]; }
```
Comment (kba) [inline js/model/line.js]: better use `let` to scope to expression.

**Comment by stweil** [comment]
@kba, I updated that code now. Do you suggest to use `let` also for the other `for` loops?

**Comment by stweil** [comment]
This pull request **fixes 8 alerts** when merging a07cd6a180b31fb6c16ffba6ab4b75cf8084f705 into 1f7191d752654ac79bdba9e5f98dca813ffd169e - [view on LGTM.com](https://lgtm.com/projects/g/UB-Mannheim/ocr-gt-tools/rev/pr-48242e42d7a36c418311dcd3f1453ee21ec7f855)

**fixed alerts:**

* 8 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by kba** [comment]
> Do you suggest to use let also for the other for loops?

For newer JS code, I would not use `var` anywhere. If you want to fix all those in one go, `eslint --fix` can [do that](https://eslint.org/docs/rules/no-var). In `for` loops it's unproblematic, in other places it might require hoisting declarations to the function scope.

**Comment by stweil** [comment]
I added a commit which uses `let` in all `for` loops.

**Comment by stweil** [comment]
This pull request **fixes 8 alerts** when merging 7c4b3095a9389f3aeb81971a56382423e9903cb2 into 1f7191d752654ac79bdba9e5f98dca813ffd169e - [view on LGTM.com](https://lgtm.com/projects/g/UB-Mannheim/ocr-gt-tools/rev/pr-99cd4c92be459a5600c807a562650bde60feaadd)

**fixed alerts:**

* 8 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Review by kba — APPROVED** [review]

**Pull request merged**